source/test/title/index.js

```diff
@@ -3,18 +3,16 @@ import reactDom from 'react-dom/server';
 import test from 'tape';
 import dom from 'cheerio';
 
-import createTitle from 'shared/components/title';
+import createTitle from 'shared/components/Title';
```

Comment:
Please use all-lowercase filenames to maximize cross-platform compat. Thanks. =)